docs/source/articles/http-caching.html.md

```diff
@@ -31,7 +31,31 @@ This creates a major complication for any data on the page with user or session
 
 **Note:** [Rack::Cache](http://rtomayko.github.io/rack-cache/) is a simple way to achieve the caching advantage explained above without the configuration and hosting burden of a more robust HTTP cache like [Varnish](https://www.varnish-cache.org).
 
-## Customizing the Rack::Cache Key
+## Using Rack::Cache
+
+Many Workarea applications use `Rack::Cache` for storing their HTTP cache. In fact, this is the caching solution that is used by [Workarea Commerce Cloud](https://www.workarea.com/pages/commerce-cloud), so if you're on Commerce Cloud, congratulations! You're already using `Rack::Cache`! 🎉
```

Comment:
This section doesn't really make sense to me - all Workarea applications use `Rack::Cache`. You would have to go out of your way to decorate it out.

Reply:
That's true. This doc sorta reads like you have a choice of using `Rack::Cache` vs. Varnish, and I would definitely like to rewrite some parts to emphasize that you get rack-cache configured in prod by default.